source/voice_control/best_practices.markdown

```diff
@@ -0,0 +1,84 @@
+
+For example, `light.living_room_lamp` might be the entity ID of `Living room lamp`. `Kitchen light` would be enough for the `light.kitchen` if you only have one light fixture in that room.
+
+Note that this convention is only a recommendation, actual naming of your devices and entities might depend on your language or personal preference.
+
+If you ever find yourself mentioning a certain device or entity in a certain way, make sure to add that as an alias, as it would probably be the most natural way to refer to the entity.
+
+Names and aliases also apply to `area`, you need to address area names and aliases in the exact same manner as you would for entities.
+
+
+### Be aware of language specificity 
+
+If you have set up Home Assistant entity names in English but plan to use Assist in another language, don’t worry. You can add aliases to each entity, allowing them to be named in any language.
+
+English has pretty simple grammar rules, but there are languages where definite articles are pre- or suffixes to words, where nouns have genders or numbers. Language leaders are making efforts to support most such declinations in each language, but they can’t control the stuff that you name. So try to think whether a certain entity having an unarticled name would be called out in a sentence requiring a definite article or viceversa. If so, add that version of the name as an alias as well.
+
+### Check domains and device classes 
+
+Assist leverages domains to define the proper verbs for the action being taken (e.g. turning on/off a `light` or a `fan`, opening/closing a `cover` with a `door` `device_class`, opening/closing a `valve`, locking/unlocking a `lock` etc.).
+
+It might not bother anyone to have a `switch.main`_valve in the UI instead of a valve, but you can’t ask Assist to open the main valve if the main valve is a switch. If it was a `valve.main_valve`, then the former sentence would have worked without a hitch.
+
+To prevent this, you can use either the [Change device type of a switch integration](/integrations/switch_as_x/) or create virtual entities using [template](/integrations/template/) entities or Generic X (e.g. [generic thermostat](/integrations/generic_thermostat/)).
+
+The same thing applies to some device classes. For example, if you have a `binary_sensor.bedroom_window` with no `device_class` set, you can only ask whether the bedroom window is on, which doesn’t even make sense. To be able to ask if it’s open, you need to set a proper `device_class` to that `binary_sensor`, i.e. window.
+
+
+## Ready?
+Once your devices and entities are correctly 
+- Exposed to assist
+- Assigned to areas.
+
+It is now time to speak to your device.
+
+For setting up your device (and have a mic and speaker to execute the voice interaction) you can either have your phone or a custom device. Check here how to do it in [Android](/voice_control/android/) and [Apple](/voice_control/apple/) devices.
```

Comment:
what is meant by "your device" here? Assist?

Reply:
the hardware you would be using for talking to/from Assist. Its true if could use a better phrasing though